default_pick_pose: [0.0, -0.7853, 0.0, -2.3561, 0.0, 1.5707, 0.7853]
default_place_pose: [-1.5708, 0.0, 0.0, -2.37365, 0.0, 2.37365, 0.0]
default_velocity_scaling: 0.7
default_acceleration_scaling: 0.7